synapse/handlers/register.py

```diff
@@ -362,70 +360,6 @@ def appservice_register(self, user_localpart, as_token):
         )
         return user_id
 
-    @defer.inlineCallbacks
-    def check_recaptcha(self, ip, private_key, challenge, response):
```

Comment:
Why are we removing this?

Reply:
There were a few functions that were not called anywhere in the codebase, and this was one of them.